src/utils/numbers.ts

```diff
@@ -7,20 +7,32 @@ type FormatCurrency = {
   currency?: string
   divisor?: number
   fractionDigits?: number
+  currencyPosition?: boolean
 }
 
-export const formatCurrency = ({ value, currency = '$', divisor = 1, fractionDigits = 2 }: FormatCurrency): string => {
+export const formatCurrency = ({
```

Comment:
Also why we are updating this function at all?

Reply:
i was testing how the UI looks with the $ change to old solana currency sign

Reply:
so we should revert this change, right?

Reply:
Isn't it better to keep it this way, it could easily come in handy? I've actually removed another currency formating function in previous PRs (and refactored the project) with the goal of us having one main currency formatting function.